vyper/codegen/module.py

```diff
@@ -2,7 +2,7 @@
 
 from typing import Any, Dict, List, Optional, Tuple
 
-from vyper import ast as vy_ast
+from vyper import ast as vy_ast, utils
```

Comment:
it's best practice (and the linter will complain otherwise) for each import to have its own statement. i would also just import the specific functions needed from vyper.utils.

Reply:
thanks!